python_modules/dagster-graphql/dagster_graphql/implementation/execution/backfill.py

```diff
@@ -67,6 +68,12 @@ def create_and_launch_partition_backfill(graphene_info, backfill_params):
         reexecution_steps=backfill_params.get("reexecutionSteps"),
         tags={t["key"]: t["value"] for t in backfill_params.get("tags", [])},
         backfill_timestamp=pendulum.now("UTC").timestamp(),
+        asset_selection=[
+            AssetKey.from_graphql_input(asset_key)
+            for asset_key in backfill_params.get("assetSelection")
+        ]
+        if backfill_params.get("assetSelection")
```

Comment:
I think it shouldn't be possible for `assetSelection` to have type None if the input param type is
```
graphene.List(graphene.NonNull(GrapheneAssetKeyInput))
```
Would it make sense to make assetSelection take in `None` to signify an asset selection wasn't provided?